projects/mercury/.eslintrc.json

```diff
@@ -5,20 +5,27 @@
     "jest/globals": true
   },
   "plugins": [
+    "ft-flow",
     "react-hooks",
     "jest"
   ],
   "extends": [
     "airbnb",
     "plugin:jest/recommended"
   ],
-  "parser": "@babel/eslint-parser",
+  "parser": "hermes-eslint",
```

Comment:
@babel/eslint-parser can be then removed from dependencies, right?

Reply:
Good point, removed. Double-checked the config one more time and cleaned it up even more